recipe/meta.yaml

```diff
@@ -24,19 +26,19 @@ requirements:
     - perl
   host:
     - python 3.6
-    - cairo
+    - cairo  # [not win]
     - curl
     - freetype
     - glib
-    - gobject-introspection 1.*
+    - gobject-introspection 1.*  # [not win]
```

Comment:
`gobject-introspection` should compile with MSVC (see https://github.com/GNOME/gobject-introspection/blob/master/MSVC.README.rst) but we need to have a more recent version with `meson` support.

Reply:
This is in flight, [gobject-introspection-feedstock #9](https://github.com/conda-forge/gobject-introspection-feedstock/pull/9). We're blocked on a new release of Meson to fix an macOS build bug (or potentially backporting the fix); [issue here](https://github.com/mesonbuild/meson/issues/4741).